### PR TITLE
LPS-111596 Render default value when field does not exist

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/floating-toolbar/CollectionConfigurationPanel.js [hunk 1]
```diff
@@ -23,7 +23,14 @@ import selectPrefixedSegmentsExperienceId from '../../selectors/selectPrefixedSe
 import {useDispatch, useSelector} from '../../store/index';
 import updateItemConfig from '../../thunks/updateItemConfig';
 
-const NUMBER_OF_COLUMNS_OPTIONS = ['1', '2', '3', '4', '5', '6'];
+const LAYOUT_OPTIONS = [
```
Comment (ealonso) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/floating-toolbar/CollectionConfigurationPanel.js:26]: it should bee 2-columns, 3-columns

Comment (ealonso) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/floating-toolbar/CollectionConfigurationPanel.js:26]: cc: @sandrodw3

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/floating-toolbar/CollectionConfigurationPanel.js:26]: Done!

**Comment by ealonso** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by ealonso** [comment]
https://github.com/liferay-echo/liferay-portal/pull/145

**Pull request closed without merging**